src/hooks/useObserver.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useRef, useState, MutableRefObject } from 'react';
+
+const useObserver = (options = {}): [MutableRefObject<HTMLElement | null>, boolean] => {
```

Comment:
```suggestion
const useObserver = (options?: IntersectionObserverInit): [MutableRefObject<HTMLElement | null>, boolean] => {
```
현재 options를 아무것도 받지 않고있는데
혹시 받을수 있으니까 이런식으로 설정해주셔도 좋을거같아요

Reply:
와오 intersectionObserverInit 첨 보네여!!! 바로 수정햇슴당